src/Maker/Factory/MakeFactoryData.php

```diff
@@ -43,6 +48,8 @@ public function __construct(
         private string $staticAnalysisTool,
         private bool $persisted,
         bool $withPhpDoc,
+        /** @var string[]|true $forceProperties */
+        array|bool $forceProperties
```

Comment:
```suggestion
        private bool $forceProperties
```

(BTW, I'm not sure, but I think that PHPStan does not understand because you should either use `@param` above the `__construct` method, or use `@var` here but without the name of the property)